test/postgresDbTest.js

```diff
@@ -0,0 +1,85 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const server = require('../server/index.js');
+const postgresDb = require('../postgresDb/index.js');
+
+const should = chai.should();
+
+chai.use(chaiHttp);
+
+const id = Math.floor(Math.random() * 300000);
+
+describe('Movies', () => {
+  // it('testing', (done) => {
+  //   chai.request('http://localhost:3000')
+  //     .get('/tetraflix/testing')
+  //     .end((err, res) => {
+  //       console.log(res.text);
+  //       res.should.have.status(200);
+  //       res.text.should.be.a('string');
+  //       res.text.should.equal('testing, testing, testing');
+  //       done();
+  //     });
+  // });
+
+  it('should contain at least 300,000 movies', () => {
+    postgresDb.movieDb.query('select count(*) from movies')
+      .then((result) => {
+        const isBigEnough = result[0][0].count >= 300000;
+        isBigEnough.should.equal(true);
+      })
+      .catch((err) => {
+        console.error(err);
+      });
+  });
+
+  it('each movie should have a title', () => {
+    // const queryString = 'select * from movies where id = ' + id;
```

Comment:
I don't think you need these lines if they're commented out and you're putting the query string directly into your query method.

Reply:
I stored the query string in a variable and now I'm only using the variable.  I deleted the commented out lines